package.json

```diff
@@ -1,50 +1,34 @@
 {
 	"name": "@guardian/consent-management-platform",
 	"version": "3.4.8",
-	"description": "Library of useful utilities for managing consent state across *.theguardian.com",
+	"description": "Consent management for *.theguardian.com.",
+	"homepage": "https://github.com/guardian/consent-management-platform.git",
+	"repository": "https://github.com/guardian/consent-management-platform.git",
+	"license": "Apache-2.0",
 	"main": "dist/index.js",
+	"module": "dist/index.esm.js",
 	"types": "dist/index.d.ts",
 	"files": [
 		"dist/**/*"
 	],
 	"scripts": {
-		"build": "yarn validate && yarn build-only",
-		"build-only": "yarn clean && rollup --config && yarn declaration",
+		"build": "npm-run-all clean --parallel compile types",
```

Comment:
Mixing NPM & Yarn?

Reply:
yeah it's a confusing name. it allows us to run scripts in parallel and serial as we choose. in this instance, `clean` runs, then `compile` and `types` are run in parallel, which i don't think yarn can do?